test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -44,5 +48,87 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace("script processed queue message")));
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace(messageContent)));
         }
+
+        protected async Task EasyTablesTest(bool writeToQueue = true)
+        {
+            // EasyTables needs the following environment vars:
+            // "AzureWebJobsMobileAppUri" - the URI to the mobile app
```

Comment:
Should be, yeah -- they use the same table but explicitly look for the GUID ids they're dealing with.

Is this how you handle EventHubs as well (have a dedicated test service set up)? There'll be more coming soon (DocDB, NH) so we should all follow the same pattern.

---
In reply to: [56191144](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56191144) [](ancestors = 56191144)